### PR TITLE
chore: deps update parser

### PR DESCRIPTION
**Pull request by evilebottnawi**
Includes:
- Change `static` ast printing
- Fixed `constantstatement` and `classconstant` printing with `value` as prettier format `const`/`let`/`var`
- Tests

Before we use same algorithm for `*constant` and `static` nodes. Now they are same in some places, but it is really difference ast, also `*constant` nodes ast will be changed in near future

### CODE REVIEW

File: src/printer.js [hunk 1]
```diff
@@ -2374,8 +2405,7 @@ function printNode(path, options, print) {
       return printAssignment(
         node.left,
         path.call(print, "left"),
-        // Assignments nested inside a static declaration doesn't have the operator set, so printing manually
-        concat([" ", node.operator ? node.operator : "="]),
+        concat([" ", node.operator]),
```
Comment (evilebottnawi) [inline src/printer.js]: Not `static` have valid ast, no need this hack

**Comment by evilebottnawi** [comment]
I hate `AppVeyor` :disappointed: cache invalidation doesn't work in many many many cases

**Comment by evilebottnawi** [comment]
@czosel any ideas what is wrong with `AppVeyor`?

File: .appveyor.yml [hunk 2]
```diff
@@ -11,14 +11,15 @@ build: off
 deploy: off
 
 cache:
-  - node_modules
-  - "%LOCALAPPDATA%/Yarn"
+  - node_modules -> appveyor.yml, package.json, yarn.lock
+  - "%LOCALAPPDATA%/Yarn -> appveyor.yml, package.json, yarn.lock"
```
Comment (evilebottnawi) [inline .appveyor.yml]: Setup cache invalidation :+1: to avoid problem in future

Comment (czosel) [reply .appveyor.yml]: nice!

**Pull request merged**